Core/include/Acts/Propagator/StraightLineStepper.hpp

```diff
@@ -131,6 +131,16 @@ class StraightLineStepper {
 
   StraightLineStepper() = default;
 
+  template <typename charge_t>
+  State makeState(std::reference_wrapper<const GeometryContext> gctx,
```

Comment:
Same as above.

Reply:
Ok. You'll look into that when this PR is merged?